charts/observability-pipelines-worker/values.yaml

```diff
@@ -0,0 +1,340 @@
+# See Observability Pipeline helm documentation to learn more:
+## https://docs.datadoghq.com/observability_pipelines/kubernetes/helm/
+
+## FOR AN EFFORTLESS UPGRADE PATH, DO NOT COPY THIS FILE AS YOUR OWN values.yaml.
+## ONLY SET THE VALUES YOU WANT TO OVERRIDE IN YOUR values.yaml.
+
+# nameOverride -- Override the name of the app.
+nameOverride: ""
+
+# fullnameOverride -- Override the fully qualified app name.
+fullnameOverride: ""
+
+# commonLabels -- Labels to apply to all resources.
+commonLabels: {}
+#  team_name: dev
+
+datadog:
+  # datadog.apiKey -- Specify your Datadog API key.
+  apiKey: <DD_API_KEY>
+  # datadog.apiKeyExistingSecret -- Specify a preexisting Secret which has your API key instead of creating a new one.
+  # The value must be stored under the `api-key`.
+  apiKeyExistingSecret: ""
+  # datadog.configkey -- Specify your Datadog Configuration key.
+  configKey: <DD_CONFIGURATION_KEY>
+  # datadog.configKeyExistingSecret -- Specify a preexisting Secret which has your Configuration key instead of
+  # creating a new one. The value must be stored under the `config-key`.
+  configKeyExistingSecret: ""
+  ## Set to 'datadoghq.com' to send data to the US1 site.
+  ## Set to 'datadoghq.eu' to send data to the EU site.
+  ## Set to 'us3.datadoghq.com' to send data to the US3 site.
+  ## Set to 'us5.datadoghq.com' to send data to the US5 site.
+  ## Set to 'ddog-gov.com' to send data to the US1-FED site.
+  # datadog.site -- The [site](https://docs.datadoghq.com/getting_started/site/) of the Datadog intake to send data to.
+  site: datadoghq.com
+
+image:
+  # image.name -- Specify image name to use (relative to `image.repository`).
+  name: observability-pipelines-worker
+  # image.tag -- Specify image tag to use.
+  tag: 0.1.0
+  # image.digest -- (string) Specify image digest to use, takes precedence over `image.tag`.
+  digest:
+  ## Currently, we offer images at:
+  ## - GCP: gcr.io/datadoghq
+  ## - DockerHub: docker.io/datadog
+  ## - AWS: public.ecr.aws/datadog
+  # image.repository -- Specify image repository to use.
+  repository: gcr.io/datadoghq
+  # image.pullPolicy -- Specify the
+  # [pullPolicy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy).
+  pullPolicy: IfNotPresent
+  # image.pullSecrets -- Specify the
+  # [imagePullSecrets](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod).
+  pullSecrets: []
+  #  - name: <REGISTRY_SECRET>
+
+# replicas -- Specify the number of replicas to create.
+replicas: 1
+
+# podManagementPolicy -- Specify the
+# [podManagementPolicy](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies).
+podManagementPolicy: OrderedReady
+
+## TODO: Support Watermark Pod Autoscaler?
+autoscaling:
+  # autoscaling.enabled -- If **true**, create a
+  # [HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/).
+  enabled: false
```

Comment:
**🌶️ Spicy Opinion Hour 🌶️**

Any particular reason we wouldn't want to enable this by default? Pretty much anyone using this is gonna want to autoscale, I'm finding.

Reply:
The only thing immediately coming to mind would be deploying to clusters where it's not supported. If it's an option that the majority of users are going to be enabling, I figure we can default it to `true` and gracefully handle the failure paths.